templates/classes/Container.js

```diff
@@ -372,46 +374,49 @@ export default class Container extends Morph {
         } else {
           setTimeout(() => {
             checkForEditor()
-          },100) 
+          },100)
         }
       };
       checkForEditor()
     })
   }
-  
+
   thumbnailFor(url, name) {
     if (name.match(/\.((png)|(jpe?g))$/))
       return "<img class='thumbnail' src='" + name +"'>"
     else
-      return ""    
+      return ""
   }
-  
+
   linksForFile(url, name) {
     if (name.match(/\.((mkv)|(mp4)|(avi))$/))
       return "<a class='play' href='" + (""+url).replace(/\/?$/,"/") + name +"'>play</href>"
     else
-      return ""    
+      return ""
   }
-  
+
   listingForDirectory(url, render) {
     return lively.files.statFile(url).then((content) => {
       var files = JSON.parse(content).contents;
-      var index = _.find(files, (ea) => ea.name.match(/^index\.md$/i)) 
-      if (!index) index = _.find(files, (ea) => ea.name.match(/^index\.html$/i))
-      if (index) { 
-        return this.setPath(url + "/" + index.name) 
-      }
+      var index = files.find((ea) => ea.name.match(/^index\.md$/i))
+
+      if (!index)
+        index = files.find((ea) => ea.name.match(/^index\.html$/i))
+
+      if (index)
+        return this.setPath(url + "/" + index.name)
+
       this.sourceContent = content
       var html = "<div class='table-container'>"+
         "<table class='directory'>"+
         "<tr><th></th><th>name</th><th>size</th></tr>" +
         // "<li><a href='../'>..</a></li>" +
-        _.sortBy(files, ea => ea.name)
-          .filter(ea => !ea.name.match(/^\./))
-          .map( ea =>
-          // "<li><a href='"+ea.name + (ea.type == "directory" ? "/" : "")+"''>" +ea.name+ "</a></li>"
-          "<tr><td>"+this.thumbnailFor(url, ea.name)+"</td><td>" + ea.name + '</td><td>' + ea.size+ '</td><td>'+this.linksForFile(url, ea.name)+'</td></tr>'
-          ).join("\n")+"</table></div>"
+        files
+          .sort((a, b) => a.name.localeCompare(b))
```

Comment:
O I like sortBy so much more... and I think this is a bug.... and it should be than 
 ` .sort((a, b) => a.name.localeCompare(b.name))`